NEWS.md

```diff
@@ -1,8 +1,19 @@
 MathOptInterface (MOI) release notes
 ====================================
 
+v0.6.1 (September 22, 2018)
+---------------------------
+
+- Fix cache used in `MOIB.fullbridgeoptimizer` (#524).
```

Comment:
Might be clearer to word this as enabling the `PositiveSemidefiniteConeSquare` bridge.